gammapy/scripts/tests/test_make_bg_cube_models.py

```diff
@@ -0,0 +1,24 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from astropy.tests.helper import pytest
+from ...datasets import get_path
+from ..make_bg_cube_models import main as make_bg_cube_models_main
+
+
+try:
+    import scipy
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
+@pytest.mark.xfail
+def test_make_bg_cube_models_main(tmpdir):
```

Comment:
Implementing at least one test for this is really important -> please make this high priority.